develop/migration_mt_django.rst

```diff
@@ -530,29 +477,18 @@ We need one final migration to tell Citus to mark tables for distribution. Creat
     ]
 
     operations = [
-      migrations.RunSQL(
-        "SELECT create_distributed_table('myapp_account','id')"
-      ),
-      migrations.RunSQL(
-        "SELECT create_distributed_table('myapp_manager','account_id')"
-      ),
-      migrations.RunSQL(
-        "SELECT create_distributed_table('myapp_project','account_id')"
-      ),
-      migrations.RunSQL(
-        "SELECT create_distributed_table('myapp_projectmanager','account_id')"
-      ),
-      migrations.RunSQL(
-        "SELECT create_distributed_table('myapp_task','account_id')"
-      ),
-
+      tenant_migrations.Distribute('Country', reference=True),
```

Comment:
It would be good to add the import for this. Might not be straight forward. 

```
from django_multitenant.db import migrations as tenant_migrations
```